packages/button/src/ButtonBase.ts

```diff
@@ -215,12 +218,16 @@ export class ButtonBase extends ObserveSlotText(LikeAnchor(Focusable), '', [
         if (changed.has('href')) {
             this.manageAnchor();
         }
-        if (changed.has('label')) {
-            this.setAttribute('aria-label', this.label || '');
-        }
+
         if (this.anchorElement) {
             this.anchorElement.addEventListener('focus', this.proxyFocus);
             this.anchorElement.tabIndex = -1;
         }
     }
+    protected override update(changes: PropertyValues): void {
+        super.update(changes);
+        if (changes.has('label')) {
+            this.setAttribute('aria-label', this.label || '');
```

Comment:
I don't think `aria-label` should ever be empty string! It becomes redundant

Reply:
Done!